packages/twenty-server/package.json

```diff
@@ -23,6 +23,7 @@
     "typeorm:migrate": "yarn typeorm migration:run -d ./src/database/typeorm/metadata/metadata.datasource.ts && yarn typeorm migration:run -d ./src/database/typeorm/core/core.datasource.ts",
     "database:init": "yarn database:setup && yarn database:seed:dev",
     "database:setup": "npx ts-node ./scripts/setup-db.ts && yarn database:migrate",
+    "database:setup:prod": "npx ts-node ./scripts/setup-db.ts && yarn typeorm:migrate",
```

Comment:
don't you also seed in this one?

Reply:
we said we should not seed. Seeds are run when create workspace so we don't need them here i beleive